paramMismatch.chpl:5: error: unresolved call 'foo(10: int(64))'
paramMismatch.chpl:1: note: candidates are: foo(x: int(32))
